Criipto.Signatures/Client.cs

```diff
@@ -0,0 +1,21 @@
+
+using GraphQL.Client.Abstractions;
+using GraphQL.Client.Http;
+using GraphQL.Client.Serializer.Newtonsoft;
+using System.Net.Http.Headers;
+
+namespace Criipto.Signatures {
+    public class CriiptoSignaturesClient : IDisposable {
+        private GraphQLHttpClient graphQLClient;
```

Comment:
Making members `private readonly` is idiomatic

Reply:
@sgryt `readonly` is settable from constructor?

Reply:
Yes, but not mutable from other members.